helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -33,11 +33,6 @@ configmap:
   # Increase hash table size to allow more server names for stability reasons
   server-name-hash-bucket-size: "1024"
 
-  # configmap.server-tokens
-  # Controls whether to send NGINX Server header in responses and display NGINX
-  # version in error pages.
-  server-tokens: "false"
```

Comment:
Now upstream defaults to server-tokens disabled, so there's no need to have this override.